startup.py

```diff
@@ -97,7 +97,16 @@ def prepare_launch(self, exec_path, args, file_to_open=None):
             "basic",
             "bootstrap.py"
         )
-
+        #To be removed when Python Package is integrated properly
+        #Code is passing a PySide2 environment variable to Shotgun unity to use for install
+        #A proper integration would simple Pip install Pyside
+        from sgtk.util.qt_importer import QtImporter
+        pyside_importer = QtImporter()._import_pyside2()
+        pysideFilePath = pyside_importer[2].__path__[0]
+        pysideFilePath=pysideFilePath.replace("Python","Python3")
+        pysideFilePath=os.path.dirname(pysideFilePath)
+        required_env["SHOTGUN_UNITY_PYSIDE_LOCATION"] = pysideFilePath
+        
```

Comment:
Why is this change showing? I thought you merged that already.
You should rebase/merge your branch to make it disappear